s3/pkg/service/object.go

```diff
@@ -362,6 +375,20 @@ func (s *s3Service) GetObject(ctx context.Context, req *pb.GetObjectInput, strea
 			break
 		}
 
+		if bucket.ServerSideEncryption.SseType == "SSE" {
+			// decrypt and write
+			fmt.Println("from cloud")
+			fmt.Println(buf[0:n])
```

Comment:
Here, use log instead of fmt please, and is it a good idea to print all the buffer?

Reply:
apologies @sfzeng, was debugging some byte padding issue locally, forgot to remove these before commit, removed now